core/templates/pages/exploration-editor-page/translation-tab/translator-overview/translator-overview.component.spec.ts

```diff
@@ -233,6 +236,101 @@ describe('Translator Overview component', () => {
         .createSpy()
         .and.returnValue(Promise.resolve(entityTranslation));
     });
+    it('should update entity translations with edit translation changes', fakeAsync(() => {
```

Comment:
Missing newline above this one. (If this gets merged before you see this comment, feel free to fix in a separate PR.)

Reply:
Done